src/components/Media/Media.js

```diff
@@ -1,9 +1,11 @@
 import { Link } from "@reach/router";
+import ButtonAppBar from "../AppBar";
 
 function Media() {
   return (
     <>
-      <Link to="/home">Home</Link>
+      <Link to="/">Home</Link>
+      <ButtonAppBar title="Media" />
```

Comment:
```suggestion
      <ButtonAppBar title="Filme & Serien" />
```
Habe mir es mal erlaubt so zu nennen :)